### ISSUE TITLE
I think the precision and recall is calculated wrong

### ISSUE THREAD

**Issue by jpatrickpark** [jpatrickpark/1003-project#12]
It should be looking at the entire estimated matrix and get the ones with the biggest estimation value, but it is only looking at the test data to generate recommendations.

**Comment by jpatrickpark** [comment]
Let's say we have 100*100 matrix, and we have 250 reference, and we split it so that we have 200 training data and 50 test data, we need a test set of size 9800 rather than 50.

**Comment by jpatrickpark** [comment]
We need to write a code to generate testset which is (entire matrix - train data). Currently testset is just the holdout data. What might be happening is that the matrix is just predicting something very close to 1 for every entry, in which case it's really not a good model.
